src/map/handler/Map.Keyboard.js

```diff
@@ -4,8 +4,7 @@
 
 L.Map.mergeOptions({
 	keyboard: true,
-	keyboardPanOffset: 80,
-	keyboardZoomOffset: 1
+	keyboardPanDelta: 80
```

Comment:
I know this is an API change. But hey, it's Leaflet 1.0 and we can make braking changes if just for the sake of variable naming consisency! :-)